hail/src/main/scala/is/hail/variant/RegionValueVariant.scala

```diff
@@ -1,56 +1,54 @@
 package is.hail.variant
 
 import is.hail.annotations._
-import is.hail.expr.types.physical.{PArray, PString, PStruct}
+import is.hail.expr.types.physical.{PArray, PBinary, PLocus, PString, PStruct}
 import is.hail.utils._
 
 class RegionValueVariant(rowType: PStruct) extends View {
```

Comment:
The use of ptypes here was wrong: reliance on static/canonical methods on PString, and lack of use of PLocus methods.